app/views/console.html

```diff
@@ -16,7 +16,7 @@
           </div>
 
 
-          <form role="form">
+          <form role="form" ng-show="online">
```

Comment:
I would not hide the console, but catch the case if someone wants to submit a query.